docs/building-assets.md

```diff
@@ -0,0 +1,98 @@
+# Building assets
+
+## Overview
+
+Toolkit provides a way to build theme assets with Gulp.js.
+
+By default a gulpfile is inclued and as well some npm packages in order to:
+
+- Look for Scss files and convert them into Css
+- Minify Css and Js
+- Merge files into one minimized file
+
+## How to use
+
+### Source files
+
+The folder structure for the source files should be aligned like this:
+
+- /theme_folder/css
+- /theme_folder/css
+- /theme_folder/js
+
+Note: It will search on all childern folders.
+
+After the task is complete the generated 'theme_folder/assets' will look like this:
```

Comment:
Specificy that assets is the default value provided. if they override a new one can be produced.